bridge/client/webrtc.js

```diff
@@ -1086,7 +1086,7 @@
 
         function findInArrayById(array, id) {
             for (var i = 0; i < array.length; i++)
-                if (array[i].id == id);
+                if (array[i].id == id)
```

Comment:
My guess is that they are.. I did not write the code.. I just fixed the problem. My guess is == can be changed to === in a lot of places. That could be done in another patch if needed I guess?